### ISSUE TITLE
Can SparK be used for few-shot learning?

### ISSUE THREAD

**Issue by Y-T-G** [keyu-tian/SparK#77]
Like SwaV, is it possible to fine-tune SparK pretrained models on a very small subset of the dataset and get good accuracy, or is it not intended for that?

**Comment by keyu-tian** [comment]
@Y-T-G Although I didn't try such a low-shot finetuning, I believe it is hopeful to get a decent accuracy. E.g., when finetuing ConvNeXt-Small, I observed the accuracies of the first three finetuning epochs of were 28.4, 53.7, and 61.9.

**Comment by Y-T-G** [comment]
@keyu-tian I see. Alright.
